### DISCUSSION TITLE
Best channels for contacting involved people?

### DISCUSSION THREAD

**Discussion by mikonieminen** [getsops/sops#1108] [Q&A]
I recently had an itch related to `exec-file` not supporting extracting only a subset of the secrets. I created an issue about it and then I realized it's actually quite easy to implement so I went and created a PR. Because I'm not experienced with Go, it's probably not perfect implementation, but at least it seems to do the job (it's practically just a copy-paste of couple of lines).

After almost three weeks, nothing happened on my PR. It is not even approved for running checks.

I thought I'll contact some of the developers. First thing I found is that the instructions in [CONTRIBUTING.md](https://github.com/mozilla/sops/blob/master/CONTRIBUTING.md#communication) are outdated and suggesting to join Mozilla's IRC `go` channel. Then I checked if I could get some hints/help on the `golang` room on Mozilla's matrix server. There I was told it's not the right place and I should try this Github Discussions instead. I was also recommended to DM @ajvb which is probably not the right thing to do.

This makes me wonder what is the right channel to ask help or ping maintainers?

**Comment by reinouts** [comment]
I would like to know as well. I am interested in https://github.com/mozilla/sops/pull/898 but the assigned reviewer has gone silent.

**Comment by onedr0p** [comment]
In https://github.com/mozilla/sops/discussions/927 @mhoye and @rforsythe have given an update but I think they are beating around the bush. We're really unsure of the project status here and what the future has in store for mozilla managing sops.
